packages/fxa-settings/src/components/Settings/MfaGuard/error-boundary.test.tsx

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MfaErrorBoundary } from './error-boundary';
+import { JwtTokenCache } from '../../../lib/cache';
+import { before } from 'node:test';
```

Comment:
Is this an IDE populated import?  I don't think that's the `before` we use with jest.

Reply:
Ah, yeah... good catch.